cafebabel/templates/articles/translations/detail.html

```diff
@@ -20,7 +20,13 @@ <h1>{{ translation.title }}</h1>
   {{ translation.body|markdown }}
 
   <p>
-    Created on <time>{{ translation.original_article.creation_date.date() }}</time> by {{ translation.author }}
+    <span>Created on <time pubdate="{{ translation.original_article.creation_date.date() }}">{{ translation.original_article.creation_date.date() }}</time></span>
+    — by {% if article.author.profile.has_image %}
```

Comment:
Tests are broken because you reference `article` here and it doesn't exist, it should be `translation.original_article` @cafebabel-webmaster